Tools/ros2/ardupilot_msgs/srv/Takeoff.srv

```diff
@@ -0,0 +1,10 @@
+
+# This service requests the vehicle to takeoff
+
+# alt : Set the takeoff altitude
```

Comment:
Can you specify the altitude datum? Is this height above terrain, height above home, or MSL?